session.go

```diff
@@ -1,97 +1,99 @@
 package bond
 
 import (
+	"database/sql"
+	"fmt"
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/lib/sqlbuilder"
```

Comment:
We're importing `sqlbuilder` here because it provides interfaces for creating SQL adapters, in fact these interfaces for SQL adapters can live anywhere, even on `db.go` the only problem is that they require importing `database/sql`, that's why I moved them away from `db.go`.